src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -34,24 +36,55 @@
 public class Helios {
 
   public static HashMap<String, String> tokenCache = new HashMap<String, String>();
+  private static RequestConfig requestConfig = RequestConfig.custom().setConnectTimeout(3000)
+      .setSocketTimeout(3000).build();
 
 
-  private Helios() {
-  }
+  private Helios() {}
 
   public static String getAccessToken(User user) {
     return getAccessToken(user.getUserName(), user.getPassword());
   }
 
+  public static void deleteAllTokens(User user) {
+    String heliosGetTokenURL = HeliosConfig.getUrl(HeliosConfig.HeliosController.USERS);
+
+    CloseableHttpClient httpClient =
+        HttpClientBuilder.create().disableCookieManagement().disableConnectionState()
+            .disableAutomaticRetries().build();
+
+    HttpDelete httpDelete =
+        new HttpDelete(String.format("%s/%s/tokens", heliosGetTokenURL, user.getUserId()));
+    httpDelete.setConfig(requestConfig);
+    httpDelete.setHeader("THE-SCHWARTZ", Configuration.getCredentials().apiToken);
+
+    CloseableHttpResponse response = null;
+    try {
+      try {
+        response = httpClient.execute(httpDelete);
+      } catch (ConnectTimeoutException e) {
+        PageObjectLogging.log("Timeout when connecting to helios", e, true);
+        response = httpClient.execute(httpDelete);
+      }
+
+      PageObjectLogging.log("DELETE HEADERS: ", response.toString(), true);
+
+    } catch (ClientProtocolException e) {
+      PageObjectLogging.log("CLIENT PROTOCOL EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+      throw new WebDriverException(e);
+    } catch (IOException e) {
+      PageObjectLogging.log("IO EXCEPTION",
+          "PLEASE CHECK IF YOUR VPN IS ENABLED" + ExceptionUtils.getStackTrace(e), false);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "PLEASE CHECK IF YOUR VPN IS ENABLED" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)